### ISSUE TITLE
[Quest] Banshee's Revenge

### ISSUE THREAD

**Issue by CitroenXara** [wow-mania/Redemption#474]
I blew the horn one time and the NPC didn't show up . After that I tried to blow it again and it didn't work . Please solve the problem or make the quest complete for me . Ingame name "Taros" from Alliance.

**Comment by Emzzibub** [comment]
Yeah same problem still. I've blown the horn and nothings happened and i can't re-click the horn either.
